<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>677</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Settings">
       <attribute name="title">
        <string>Начальные настройки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="Population_number">
          <item>
           <widget class="QLabel" name="Number_people">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Численность населения:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_people"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="Number_of_companies">
          <item>
           <widget class="QLabel" name="Number_firm">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Количество компаний:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_firm"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="OTRASL">
          <item>
           <widget class="QLabel" name="Number_otrasl">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Количество отраслей:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_otrasl"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="Cost_trud">
          <item>
           <widget class="QLabel" name="Coast_trud">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Стоимость труда за час:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_trud"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QVBoxLayout" name="KON">
          <item>
           <widget class="QLabel" name="Potrebnost">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Перечень товаров конечного потребления:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_error_product">
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;
text-decoration: underline;</string>
            </property>
            <property name="text">
             <string>NAN</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="table_kon"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="TRUD_HOUR">
          <item>
           <widget class="QLabel" name="trud_hour">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Рабочий день 1 человека:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_trud_hour"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="TRUD_HOUR_2">
          <item>
           <widget class="QLabel" name="trud_itog_2">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Количество требуемых часов для КОН:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_trud_itog"/>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="trud_itog_3">
          <item>
           <widget class="QLabel" name="nalog_all">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Количество работников:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_nalog_all"/>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="nalog">
          <item>
           <widget class="QLabel" name="trud_itog_6">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Налог на сумму:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_trud_itog_3"/>
          </item>
         </layout>
        </item>
        <item row="9" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>56</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Main">
       <attribute name="title">
        <string>Главная Страница</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="table_main"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MOP">
       <attribute name="title">
        <string>Расчет МОП</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QTableWidget" name="table_mop">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>451</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="table_price">
            <property name="minimumSize">
             <size>
              <width>980</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>РАСЧЕТ МОП</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
